shotover-proxy/examples/cassandra-passthrough/docker-entrypoint.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+set -e
```

Comment:
Can you also add a comment at the top describing how this differs from the default file?

Reply:
Addressed this btw